scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -414,13 +414,13 @@ def motr_config_k8(self):
         raise MotrError(errno.EINVAL, "libfabric is not up.")
 
     # Update motr-hare keys only for storage node
-    if self.node['type'] == 'storage_node':
+    if (self.node['type'] == 'storage_node' or self.node['type'] == 'data_node'):
         update_motr_hare_keys(self, self.nodes)
 
     execute_command(self, MOTR_CONFIG_SCRIPT, verbose = True)
 
     # Update be_seg size only for storage node
```

Comment:
update comment to say storage or data nodes.

Reply:
comment on line 422 updated